src/Driver.cpp

```diff
@@ -194,6 +194,17 @@ void Driver::openURI(std::string const& uri)
     { // UDP udp://hostname:remoteport
         if (marker == string::npos)
             throw std::runtime_error("missing port specification in udp:// URI");
+
+        string::size_type second_marker = device.find_last_of(":");
+        if (second_marker != string::npos)
+        {
+            int remote_port = 0;
```

Comment:
`int remote_port = <actual value>` there initializing it to assign it one line later is just noise.